sdk/quantum/Microsoft.Azure.Management.Quantum/src/Generated/Models/SkuDescription.cs

```diff
@@ -33,18 +33,23 @@ public SkuDescription()
         /// </summary>
         /// <param name="id">Unique sku id.</param>
         /// <param name="name">Display name of this sku.</param>
+        /// <param name="version">Display name of this sku.</param>
         /// <param name="description">Description about this sku.</param>
+        /// <param name="restrictedAccessUri">Uri to subscribe to the
+        /// restricted access sku.</param>
         /// <param name="targets">The list of targets available for this
         /// sku.</param>
         /// <param name="quotaDimensions">The list of quota dimensions for this
         /// sku.</param>
         /// <param name="pricingDetails">The list of pricing details for the
         /// sku.</param>
-        public SkuDescription(string id = default(string), string name = default(string), string description = default(string), IList<string> targets = default(IList<string>), IList<QuotaDimension> quotaDimensions = default(IList<QuotaDimension>), IList<PricingDetail> pricingDetails = default(IList<PricingDetail>))
+        public SkuDescription(string id = default(string), string name = default(string), string version = default(string), string description = default(string), string restrictedAccessUri = default(string), IList<string> targets = default(IList<string>), IList<QuotaDimension> quotaDimensions = default(IList<QuotaDimension>), IList<PricingDetail> pricingDetails = default(IList<PricingDetail>))
```

Comment:
This is a breaking change: not adding new parameters `version` and `restrictedAccessUri` at the end.

Please rearrange the order of the parameters in the swagger, so both are added at the end.

Reply:
Good catch!

@JonathanCrd, the SKUDescription is never created by the user and there are no operations for the user to create, update or delete SKUs. The can only be fetched/read by the user as a return from the GetOfferings operation.
So the user will never instantiate them, only the generated code deserializer will do.

That's why we don't consider it a breaking change.

Please let me know if we can proceed as-is.

Reply:
Got it. Based on that, there shouldn't be issues for the customers.
I'm going to approve this PR.